### DISCUSSION TITLE
Grace Hopper on Letterman and Other Female Inventor Stuffz

### DISCUSSION THREAD

**Discussion by ardisrtilson** [nss-day-cohort-47/C47-Discussions#31] [Topix.random]
This is that Grace Hopper TV appearance Brenda was just talking about:

https://www.dailymotion.com/video/x35dsz7

Also, if you like reading, here's a book I liked about another super interesting female inventor:

https://www.penguinrandomhouse.com/books/209482/hedys-folly-by-richard-rhodes/

**Comment by Chadwell-Clark** [comment]
I think I saw this when it aired. What an awesome woman!
